app/src/main/java/ir/jaamebaade/jaamebaade_client/view/components/Navbar.kt

```diff
@@ -49,163 +51,163 @@ val routeMap = mapOf(
-                        stringResource(R.string.INTRO_SETTING_DESC)
+        Row(
+            modifier = Modifier
+                // TODO change color
+                .fillMaxWidth()
+                .height(106.dp)
+                .background(color = MaterialTheme.colorScheme.surface)
+                .padding(horizontal = 8.dp),
+            horizontalArrangement = Arrangement.SpaceEvenly,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            navbarItems.forEachIndexed { index, item ->
+                val isSelected = currentRoute == item.route.toString()
+                NavbarItem(
+                    route = item.route,
+                    currentRoute = currentRoute,
+                    iconId = item.getIcon(isSelected = isSelected),
+                    contentDescription = stringResource(item.contentDescriptionResId),
+                    isSelected = isSelected,
+                    navController = navController,
+                    modifier = Modifier.introShowCaseTarget(
+                        index = index,
+                        style = ShowcaseStyle.Default.copy(
+                            backgroundColor = MaterialTheme.colorScheme.primary,
+                            backgroundAlpha = 0.98f,
+                            targetCircleColor = MaterialTheme.colorScheme.onPrimary
+                        ),
+                        content = {
+                            ButtonIntro(
+                                stringResource(item.introTitleResId),
+                                stringResource(item.introDescResId)
+                            )
+                        }
                     )
-                }
-            )
-        )
+                )
 
+            }
+        }
     }
 }
 
 @Composable
 fun NavbarItem(
     route: AppRoutes,
     currentRoute: String?,
-    idleIcon: ImageVector,
-    selectedIcon: ImageVector,
+    iconId: Int,
     contentDescription: String,
+    isSelected: Boolean,
     navController: NavController,
     modifier: Modifier = Modifier
 ) {
-    val isSelected = currentRoute == route.toString()
-    IconButton(
+    Button(
         modifier = modifier,
+        colors = ButtonDefaults.buttonColors(
+            containerColor = Color.Transparent,
+        ),
+        contentPadding = PaddingValues(0.dp),
+        shape = RectangleShape,
         onClick = {
             var myInclusive = false
             if (currentRoute != route.toString()) {
                 if (route == AppRoutes.DOWNLOADABLE_POETS_SCREEN)
                     myInclusive = true
                 navController.navigate(route.toString(), navOptions {
-                    popUpTo("downloadedPoetsScreen") {
+                    popUpTo(AppRoutes.DOWNLOADABLE_POETS_SCREEN.toString()) {
                         inclusive = myInclusive
                     }
                 })
             }
 
         },
     ) {
-        Box(
+        Column(
```

Comment:
Why to use Column?

Reply:
because there is now a TextBox below the icon in navbar